### DISCUSSION TITLE
[ FF7 ] Character models de-sync on FMVs

### DISCUSSION THREAD

**Discussion by tekkensora** [julianxhokaxhiu/FFNx#221] [Q&A] [answered]
I'm playing the game on 7th Heaven with satsuki HD mods and 1 more extra mod (it only affects the original spanish translation of the game so no impact on FMVs) 

thing is, every FMV (for example when you meet Cid and you run away with the plane) the FMV shows the characters getting onto the plane, but once the plane moves away Cloud and the rest haven't got on it yet, another example is that it also happens on scenes where an elevator is moving and the characters are moving too, they are literally floating a few meters away from where they are supposed to be, it's like a de-sync with the FMV and the models

any idea why is this happening or how to fix it?
I'll post a picture with the mods I'm actually using in case it helps, if you need any other file in specific just say it and I'll send it to you guys
(even if it says "Qhimm Modding community" i selected the "Satsuki mod" option)
![image](https://user-images.githubusercontent.com/80450903/111137835-8db93c80-857f-11eb-9dda-60d8a16c76f7.png)

![image](https://user-images.githubusercontent.com/80450903/111138096-d670f580-857f-11eb-8485-477e0cf200b1.png)

**Comment by julianxhokaxhiu** [comment, marked as answer]
Use the official Satsuki Packs distributed by him: http://forum.tsunamods.com/viewtopic.php?f=69&t=39

**Reply by tekkensora** [reply]
yes I'm already using the "Download SYW V5.20 iros files for 7heaven" 
or you mean I should use the steam autoinstalation? it gave me the save file glitch so I went to use 7th Heaven

**Reply by julianxhokaxhiu** [reply]
If you're using 7th Heaven, I'd suggest you using the latest beta available, that we're maintaining in the Tsunamods Team, and from there you can easily update to the latest FFNx canary, which will fix your own issue. If you did install it manually, then feel free to download and install the latest canary, using the same exact steps you used :)
